lib/data-serializer.js

```diff
@@ -1,8 +1,5 @@
 'use strict'
 
-const Poller = require('./poller')
```

Comment:
just addressing lint error here and in the line below

Reply:
Is there a way we can run the linter on pull requests?

Reply:
good idea, i'll look into that for a follow up pr